packages/eslint/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @webpack-blocks/eslint - Changelog
+
+## Next release
```

Comment:
In this case we can enter the version number already, since we will publish right away.